.gitattributes

```diff
@@ -0,0 +1,3 @@
+* text=auto eol=lf
+
+*.{png,jpg,jpeg,gif,webp,woff,woff2} binary
```

Comment:
Missing "EOF" here, as suggested by the red sign. Maybe configure your IDE so that a new line at the end of a file is ensured.

Reply:
/cc @iyear I've fixed it. And please remember to add new line at the end of any files next time.

Reply:
054c85bd06d02ae290913706ce8ecbfc3834e5c8